packages/kubernetes-1.21/credential-provider-config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: CredentialProviderConfig
+{{! This isn't right as the condition is more than just whether `providers` is "truey" but whether any are enabled }}
+{{#if settings.kubernetes.credential-providers.providers}}
```

Comment:
Would it make sense to have a handlebars helper that iterates through the list:

```handlebars
{{#if (any-enabled settings.kubernetes.credential-providers.providers)}}
```

Reply:
I wasn't sure about adding a helper that would be specific to one use case like this. But I think we probably need to do that in order to properly format the yaml.